src/styles/_catalog.scss

```diff
@@ -1,22 +1,18 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
I don't know how necessary - but according to the task you should have used Media queries
for different screens - different.

everything looks good, you made it easier)